src/main/java/nl/thehyve/ocdu/validators/clinicalDataChecks/DataTypeCrossCheck.java

```diff
@@ -79,13 +80,13 @@ private boolean matchType(String value, String expectedType) {
         if (expectedType == null || expectedType.equals(TEXT_DATA_TYPE)) {
             return true;
         } else if (expectedType.equals(INTEGER_DATA_TYPE)) {
-            return isInteger(value);
+            return UtilChecks.isInteger(value);
```

Comment:
(hint for the future, no need to change)This way of calling a static method is fine, but you can also do a static import, then you can skip class name when referencing a static member of  a class. When you type a public static method name from a different class in IntelliJ it should suggest you to do a static import.